src/services/notifications.service.js

```diff
@@ -323,7 +327,10 @@ const initialiseNotificationsStatus = (emailsToSend) => {
   for (let index in emailsToSend) {
     notifications.push({
       time: undefined,
-      sent: false,
+      sent:
+        emailsToSend.length > 1 && emailsToSend[index].type === "RNG_PENDING" // In order to prevent RN pending emails from being sent to FBO when RNG service has been resolved
+          ? true
+          : false,
```

Comment:
Would be better to fix the logic so that this (and other emails we don't want to resend) are never in the emailsToSend list, but perhaps that can be done separately.

Reply:
It may be worthwhile to simplify all logic, but in a separate task